Libraries/DirectMLX.h

```diff
@@ -4114,6 +4174,10 @@ namespace dml
             OperatorNode node = {};
             node.op = std::move(op);
             node.inputs.assign(inputs.begin(), inputs.end());
+            if (!m_name.empty())
+            {
+                node.name = m_name.c_str();
```

Comment:
[minor] Why not just `node.name = m_name;`? (then it doesn't need to first scan the string for the string length again)

Reply:
I was thinking of DML_GRAPH_NODE_DESC::name, which is a `const char*` field. Whoops!